### PR TITLE
Header

### PR DESCRIPTION
**Pull request by swapnil2308**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	<body>
+		<header>
+			<div class='container'>
+				<div class='row'>
+					<a href=""><div class='logo col-md-3'>Black Sea</div>
+					</a>
+					<div class='nav navbar col-md-9'>
+						<ul class='pull-right'>
+
+							<li><a href="#">HOME</a></li>
```
Comment (yogesh-webonise) [inline index.html]: > Instead of **#** URL use **javascript:void();**
> 
> ``` javascript
> <li><a href="javascript:void(0);">HOME</a></li>
> ```

**Comment by yogesh-webonise** [comment]
> Add proper commenting with markdowns.
> - [x] Title
> - [x] Issue
> - [x] Solution

## Link of markdown

[markdown](https://guides.github.com/features/mastering-markdown/)

**Comment by swapnil2308** [comment]
# Updated

### a href="javascript:void(0)"

###### any more Changes?

@yogesh-webonise

**Comment by yogesh-webonise** [comment]
looking fine.
**Nice job.**
Markdown and PR comment is mandatory for PR from next PR.

**Comment by swapnil2308** [comment]
# Header ready for Merge pull request

[x] all issues resolved

**Comment by ajinkyaa** [comment]
@swapnil2308 : convert all in the two spaces indentation

**Comment by yogesh-webonise** [comment]
@swapnil2308  @ajinkyaa 
Where is merge PR button?

**Comment by ajinkyaa** [comment]
@yogesh-webonise : why are you merging ? my comment is not resolved yet

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+.logo{
+	font-family: 'Cherry Swash', sans-serif;
+	font-size: 40px;
+}
+header ul{
+	margin-top: 20px;
+}
+header ul li{
+	list-style-type: none;
```
Comment (ajinkyaa) [inline css/main.css]: give this property to the UL.... its 'list-style' only

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	display: inline-block;
+	margin: 0 10px;
+}
+header ul .slash{
+	border-right: 1px solid white;
+	transform: rotate(25deg);
+
```
Comment (ajinkyaa) [inline css/main.css]: why extra space?

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+header a{
+	color: #f0f0f0;
+	font-size: 18px;
+	font-family: 'Open Sans Condensed', sans-serif;
+
```
Comment (ajinkyaa) [inline css/main.css]: why extra space?

File: css/responsive.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+}
+@media screen and (max-width: 768px) {
+  header li{
+    display: block;
+  }
+  .slash{
+    display: none;
+  }
+}
+@media screen and (max-width: 767px){
```
Comment (ajinkyaa) [inline css/responsive.css]: why there are 2 max width 767?

**Pull request merged**